code/modules/mining/mine_items.dm

```diff
@@ -192,3 +192,56 @@
 	icon_opened = "miningcaropen"
 	icon_closed = "miningcar"
 
+/*********************Mob Capsule*************************/
+
+/obj/item/device/mobcapsule
+	name = "lazarus capsule"
+	desc = "It allows you to store and deploy lazarus-injected creatures easier."
+	icon = 'icons/obj/mobcap.dmi'
+	icon_state = "mobcap0"
+	w_class = 1.0
+	throw_range = 20
+	var/mob/living/captured = null
+	var/colorindex = 0
+
+/obj/item/device/mobcapsule/attack(var/atom/A, mob/user, prox_flag)
+	if(!istype(A, /mob/living/simple_animal))
+		return ..()
+	capture(A, user)
+	return 1
+
+/obj/item/device/mobcapsule/proc/capture(var/mob/target, var/mob/U as mob)
+	var/mob/living/simple_animal/T = target
+	if(captured)
+		U << "<span class='notice'>Capture failed!</span>: The capsule already has a mob registered to it!"
+		return
+	else
+		if(istype(T) && "neutral" in T.faction)
+			T.forceMove(src)
+			T.name = "[U.name]'s [initial(T.name)]"
+			T.cancel_camera()
+			name = "Lazarus Capsule: [initial(T.name)]"
+			U << "<span class='notice'>You placed a [T.name] inside the Lazarus Capsule!</span>"
+			captured = T
+		else
+			U << "You can't capture that mob!"
+
+/obj/item/device/mobcapsule/throw_impact(atom/A, mob/user)
+	..()
+	if(captured)
+		dump_contents(user)
+
+/obj/item/device/mobcapsule/proc/dump_contents(mob/user)
+	if(captured)
+		captured.loc = src.loc
```

Comment:
some mobs emit light, so you're going to have to forcemove them out

captured.forceMove(get_turf(src))